numpy/distutils/fcompiler/nv.py

```diff
@@ -0,0 +1,52 @@
+# https://developer.nvidia.com/hpc-sdk
+import sys
+
+from numpy.distutils.fcompiler import FCompiler
+
+compilers = ['NVHPCFCompiler']
+
+"""
+Since august 2020 the NVIDIA HPC SDK includes the compilers formely known as The Portland Group compilers.
+https://www.pgroup.com/index.htm
+"""
+class NVHPCFCompiler(FCompiler):
```

Comment:
Let's make this a docstring
```suggestion
import sys

from numpy.distutils.fcompiler import FCompiler

compilers = ['NVHPCFCompiler']

class NVHPCFCompiler(FCompiler):
   """ NVIDIA High Performance Computing (HPC) SDK Fortran Compiler
   
   https://developer.nvidia.com/hpc-sdk
   
   Since august 2020 the NVIDIA HPC SDK includes the compilers formerly known as The Portland Group compilers,
   https://www.pgroup.com/index.htm.
   See also `numpy.distutils.fcompiler.pg`.
```